sql/catalyst/src/test/scala/org/apache/spark/sql/connector/InMemoryTableCatalog.scala

```diff
@@ -181,9 +181,21 @@ class InMemoryTableCatalog extends BasicInMemoryTableCatalog with SupportsNamesp
 
   override def dropNamespace(namespace: Array[String]): Boolean = {
     listNamespaces(namespace).foreach(dropNamespace)
-    listTables(namespace).foreach(dropTable)
```

Comment:
I think the change is good but to clarify, @MaxGekk, is this test only PR? all changes are in tests.

Reply:
@HyukjinKwon You are right, this is test only PR. I changed PR's title and description.